include/kamping/p2p/try_recv.hpp

```diff
@@ -33,10 +33,10 @@
 #include "kamping/result.hpp"
 #include "kamping/status.hpp"
 
-///// @addtogroup kamping_p2p
+/// @addtogroup kamping_p2p
 /// @{
 
-// @brief Receives a message if one is available.
+/// @brief Receives a message if one is available.
```

Comment:
This should fix the rendering issue currently visible in https://kamping-site.github.io/kamping/group__kamping__p2p.html#ga94ea002a1aa9b85f043603b382293295

Other C++ functions with the same rendering issues have also been addressed.